src/pages/our_ambassadors.astro

```diff
@@ -547,4 +547,7 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
       </AmbassadorCard>
     </div>
   </div>
+
+  <h1>Alumni</h1>
+  Soon
```

Comment:
```suggestion
```

Reply:
I'd leave this out until you have alimni. "Soon" or "Coming soon..." feels a little anticapatory.

Reply:
Ok!